src/main/scala/com/github/tarao/slickjdbc/interpolation/Interpolation.scala

```diff
@@ -5,7 +5,7 @@ package interpolation
 import util.NonEmpty
 import scala.language.implicitConversions
 import slick.jdbc.SQLActionBuilder
-import slick.profile.SqlAction
+import slick.sql.SqlAction
```

Comment:
Package structure of slick seems to be changed with [this commit](https://github.com/slick/slick/commit/e46a0a530a8bb5581bf10a0429bcff0e8d791c04).